internal/http/github.go

```diff
@@ -3,17 +3,21 @@ package http
 import (
 	"encoding/json"
 	"fmt"
+	"golang.org/x/mod/semver"
 	"net/http"
 	"strings"
 	"time"
-
-	"golang.org/x/mod/semver"
 )
 
-// DefaultGitHub is a preconfigured instance of GitHub.
-var DefaultGitHub = GitHub{
-	HTTPClient: &http.Client{Timeout: 2 * time.Second},
-	URL:        "https://api.github.com",
+// NewGitHub returns a preconfigured instance of GitHub.
+func NewGitHub() GitHub {
```

Comment:
This have been wrapped into a function to avoid failure when `HTTP_PROXY` is malformed.